src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonManager.cs

```diff
@@ -696,53 +696,55 @@ private void InitializeFSM()
                     else
                         return GoTo(ClusterSingletonState.Younger).Using(new YoungerData(initialOldestState.Oldest));
                 }
-                else return null;
+
+                return null;
             });
 
             When(ClusterSingletonState.Younger, e =>
             {
-                DelayedMemberRemoved removed;
-                YoungerData youngerData;
-                if (e.FsmEvent is OldestChangedBuffer.OldestChanged && (youngerData = e.StateData as YoungerData) != null)
+                if (e.FsmEvent is OldestChangedBuffer.OldestChanged oldestChanged
+                    && e.StateData is YoungerData youngerData)
```

Comment:
Nice, didn't realize you could do `var` binding here the same way you can with `switch`. C# 7 is a godsend.